client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.java

```diff
@@ -81,20 +81,29 @@ public ConfigurationBuilder() {
       this.nearCache = new NearCacheConfigurationBuilder(this);
    }
 
+   /**
+    * @inheritDoc
```

Comment:
Is this somehow handled by JavaDoc customizations infinispan has in place?

If not, then its wrong, the tag is `{@inheritDoc}` and also its just meaningless noise, since JavaDoc is automatically copied.